tests/test_002_CPCandOptimizer.py

```diff
@@ -26,10 +26,7 @@
 
 #
 
-try:
-    market_df = pd.read_csv("_data/NBTEST_002_Curves.csv.gz")
-except:
-    market_df = pd.read_csv("fastlane_bot/tests/nbtest/_data/NBTEST_002_Curves.csv.gz")
+market_df = pd.read_csv("tests/_data/NBTEST_002_Curves.csv.gz")
```

Comment:
Better use general utility function `read_csv_file` (in file `utils.py`).
The reason being, it would make it easy to apply a change to all similar calls, in a single place.